src/sync/pull.rs

```diff
@@ -141,7 +152,10 @@ pub async fn begin_pull(
     let commit_hash = db_write.commit(SYNC_HEAD_NAME).await.map_err(CommitError)?;
 
     Ok(BeginTryPullResponse {
-        http_request_info: pull_resp.http_request_info,
+        http_request_info: HttpRequestInfo {
+            http_status_code: 200,
```

Comment:
This makes me kind of uncomfortable. I would rather take the info from the pull...

Reply:
Me too. I'm going to use a tuple.